index.html

```diff
@@ -53,7 +53,7 @@ <h3>Set Range</h3>
             <div class="guess-button-cont">
               <button class="bold disabled" disabled id="submit-guess">SUBMIT GUESS</button>
               <button class="bold disabled" disabled>RESET GAME</button>
-              <button class="bold disabled" disabled>CLEAR FORM</button>
+              <button class="bold disabled" disabled id="clear-form" onclick="return false;">CLEAR FORM</button>
```

Comment:
Is there a way we can integrate this click event into the js file? Ideally we can avoid writing our javascript in the html file.

Reply:
I used event.preventDefault() within the event listener in my current feature work, and that seemed to work well.